ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml

```diff
@@ -839,6 +845,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: azure-ipi-fullyprivate-proxy-arm-f28-ota
```

Comment:
I think we can prepare a dedicated OTA yaml file in ci-operator/config/openshift/verification-tests/ folder, apart from the regular ci jobs.

Reply:
hmm, then there will be 4.18/4.17/.. new yaml files added and maintained. I rmb you suggested to move ota dedicated job to general ci before in https://github.com/openshift/release/pull/55177 considering less maintenance work from OTA side, so any other concern I missed?

Reply:
That is because https://github.com/openshift/release/pull/55177 indeed run some ota e2e testing, we migrated them because the regular e2e ci jobs does not meet OTA team requirement.

In this job, it is only run some specific OTA check point. And in this PR, we have to introduce `release.target`, it would increase some maintain work for the specific test case.

Or maybe we can directly add this job into openshift-verification-tests-master__installation-nightly-4.N.yaml, no change on the existing regular ci yaml, and no need new yaml file.

Reply:
Make sense, I'm fine to add it into installation yaml files, let me update, thanks

Reply:
done